superset-frontend/src/dashboard/components/gridComponents/Tabs.jsx

```diff
@@ -35,8 +35,6 @@ import { NEW_TAB_ID, DASHBOARD_ROOT_ID } from '../../util/constants';
 import { RENDER_TAB, RENDER_TAB_CONTENT } from './Tab';
 import { TAB_TYPE } from '../../util/componentTypes';
 
-const MAX_TAB_COUNT = 10;
```

Comment:
we used to limit number of tabs in dashboard, to avoid tabs wrapping. AndD has better solution to manage this. remove limit!